videoembedutility/twigextensions/VideoEmbedUtilityTwigExtension.php

```diff
@@ -78,17 +78,32 @@ private function endsWith($haystack, $needle) {
 		}
 		
 		public function videoEmbed($input, $options = array()) {
+			$width = '100%';
+			$height = '168';
```

Comment:
Oops, my mistake...I meant that to be the same as the previous default, just got it wrong.